src/app/views/app-sections/StatusMessages.tsx

```diff
@@ -4,7 +4,10 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+    isMultiline={false}
+    onDismiss={actions.clearQueryStatus}
+    dismissButtonAriaLabel='Close'
```

Comment:
how does a screen handle this on a localized page?

Reply:
The screen reader is able to read it as well on a localized page

Reply:
yes but does it read the english word or is it localized.

Reply:
It's localized